controllers/utils/constants.go

```diff
@@ -12,7 +12,8 @@ const (
 	IBUName     string = "upgrade"
 	IBUFilePath string = common.LCAConfigDir + "/ibu.json"
 
-	ManualCleanupAnnotation string = "lca.openshift.io/manualCleanupDone"
+	ManualCleanupAnnotation    string = "lca.openshift.io/manualCleanupDone"
+	TriggerReconcileAnnotation string = "lca.openshift.io/triggerReconcile"
```

Comment:
I wonder if we should start converging to the recommended syntax guide? https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

`lca.openshift.io/trigger-reconcile`?

Reply:
Is the use of camel case discouraged? It doesn't seem like these values are breaking the syntax described at that page. Capitals are allowed, for example

Reply:
Yeah, capticals are definitely allowed. 
See examples of k8s annotations using this format https://kubernetes.io/docs/reference/labels-annotations-taints/#scheduleralphakubernetesio-defaulttolerations.

I think the guide states that `The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.` means you could have "-"," _", "." in between but not mandatory. 
Even the example it provides doesn't have '-' in its name.
  annotations:
    imageregistry: "https://hub.docker.com/"

Reply:
For sure capital is allowed. 

Some more on this
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions 

```
Other advice regarding use of labels, annotations, taints, and other generic map keys by Kubernetes components and tools:

Key names should be all lowercase, with words separated by dashes instead of camelCase
For instance, prefer foo.kubernetes.io/foo-bar over foo.kubernetes.io/fooBar, prefer desired-replicas over DesiredReplicas
```

Reply:
I have no issue with using camelCase here, myself. It aligns with the variable naming conventions and doesn't break any rules. The note about using dashes instead of camel case is "advice", seeming to be suggested as possible guidelines for a subsystem, but that it's left to users to define their own rules. It seems odd to me to have a different convention for the annotation naming vs variables, almost opposing conventions. But I have no issues with whichever way people want to go.